_posts/en/newsletters/2019-11-20-newsletter.md

```diff
@@ -108,7 +108,17 @@ summarizes recent changes to popular Bitcoin infrastructure projects.
 *In this monthly feature, we highlight interesting updates to Bitcoin
 wallets and services.*
 
-FIXME
+- **Bitfinex bech32 send support:** In a [recent blog post]
+[bitfinex bech32 blog], Bitfinex has announced support on their exchange for
```

Comment:
For the link `[recent blog post]\n[bitfinex bech32 blog]`, separating the title from the link reference with whitespace works with our Markdown processor, but I don't think that's universally supported and, perhaps more importantly, [one of our tests](https://github.com/bitcoinops/bitcoinops.github.io/blob/1acc7ec88fbd9f95e3ba30cc054f4970fe4f71f0/Makefile#L34) for broken links expects reference style links to have the `][` part of the string not to have any whitespace (but you can otherwise line break the string however you'd like).

I pushed a quick edit for this.